tests/test_utilities.py

```diff
@@ -227,3 +229,179 @@ def test_get_course_jobs(self):
 
             # verify that job is associated with correct skill
             assert job_skill.skill == course_skills.skill
+
+    @mock.patch('taxonomy.utils.translate_text')
+    def test_get_translated_course_description_with_updated_description_and_eng_lang(self, translate_mocked):
+        """
+        Validate that `get_translated_course_description` updates Translation object if
+         course description changes. Also verify that if source language is ENGLISH,
+         than keep the original text in source and translated text field.
+        """
+        existing_course_description = "abc def"
+        existing_translation = "translated text old"
+
+        new_course_description = "ghi jkl"
+        new_translation = "translated text new"
+        translate_mocked.return_value = {'SourceLanguageCode': ENGLISH, 'TranslatedText': new_translation}
+        factories.TranslationFactory(source_record_identifier=COURSE_KEY,
+                                     source_model_field='full_description',
+                                     source_model_name='Course',
+                                     source_text=existing_course_description,
+                                     translated_text=existing_translation,
+                                     translated_text_language=ENGLISH,
+                                     source_language=ENGLISH,
+                                     )
+
+        expected_translated_description = utils.get_translated_course_description(COURSE_KEY, new_course_description)
+        translation_record = Translation.objects.filter(source_model_name='Course',
```

Comment:
same as above, indentation need to be updated according to https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines